src/NextGen/DonationForm/resources/registrars/templates/elements/DonationSummary.tsx

```diff
@@ -10,7 +10,9 @@ export default function DonationSummary() {
     const amount = useWatch({name: 'amount'});
     const donationType = useWatch({name: 'donationType'});
     const period = useWatch({name: 'subscriptionPeriod'});
-    const humanReadablePeriod = `${period}ly`.toLowerCase().replace(/\w/, (firstLetter) => firstLetter.toUpperCase());
+    const humanReadablePeriod = `${period === 'day' ? 'dai' : period}ly`
```

Comment:
resolved e486d740

may be overkill but I was having fun with it :neckbeard:

Reply:
Cool. Is the singular/plural value being used anywhere? Something we might need to consider is how we are going to handle capitalization. The adjective is easy, since we only use it as a label, but the singular/plural are usually a variable in another string, so it may not be important.

Reply:
I think just doing that little replace trick as is needed should be fine - since the string would already be translated, we can dynamically capitalize it.

Reply:
Sounds good to me.